src/features/auth/components/Button.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface ButtonPropTypes {
+  status?: 'normal' | 'confirm';
+  children: React.ReactNode;
+}
```

Comment:
다음과 같이 선언해 보시는 건 어떨까요?? 🙂

```suggestion
export interface ButtonPropTypes extends React.ButtonHTMLAttributes<HTMLButtonElement>{
  status?: 'normal' | 'confirm';
  children: React.ReactNode;
}
```

참, 컴포넌트의 props 타입도 접미사를 통일시키면 좋을 거 같습니다 🙂👍